src/AppBundle/Entity/Post.php

```diff
@@ -180,4 +193,56 @@ public function setSummary($summary)
     {
         $this->summary = $summary;
     }
+
+    /**
+     * Get tags
+     *
+     * @return Tag[]|Collection
+     */
+    public function getTags()
+    {
+        return $this->tags;
+    }
+
+    /**
+     * Add tag
+     *
+     * @param Tag $tag
+     *
+     * @return Post
+     */
+    public function addTag(Tag $tag)
+    {
+        if (!$this->hasTag($tag)) {
+            $this->tags->add($tag);
+        }
+
+        return $this;
+    }
+
+    /**
+     * Remove tag
+     *
+     * @param Tag $tag
+     *
+     * @return Post
+     */
+    public function removeTag(Tag $tag)
+    {
+        if (!$this->hasTag($tag)) {
+            $this->tags->removeElement($tag);
```

Comment:
@versh23, no. It should be just `$this->tags->removeElement($tag);`, without any checkings because the `removeElement` function doesn't throw an exception when the element doesn't exist.